### PR TITLE
uast: augmented assignment roles simplified

### PR DESCRIPTION
**Pull request by juanjux**
Removes AugmentedAssignmentVariable and AugmentedAssignmentValue and documents to use the normal Assignment versions. From #61 .

### CODE REVIEW

**Comment by smola** [comment]
Let's discuss next week about this.
See https://github.com/bblfsh/sdk/issues/61

**Comment by juanjux** [comment]
I did this because of #61 :smile: but ok let's talk about these together.

File: uast/role_string.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Code generated by "stringer -type=Role"; DO NOT EDIT.
+// Code generated by "stringer -type=Role"; DO NOT EDIT
```
Comment (ajnavarro) [inline uast/role_string.go]: :rofl:

Comment (ajnavarro) [inline uast/role_string.go]: Maybe is better keep this file as it is.

Comment (juanjux) [inline uast/role_string.go]: I've not the mininal idea of how that could have changed but it will be rewritten on the next go generate anyway

**Comment by juanjux** [comment]
Closing, we'll use the BIP process to request new roles.

**Pull request closed without merging**